.maintain/node-template-release/src/main.rs

```diff
@@ -233,20 +250,30 @@ fn main() {
 		// Check if this is the top level `Cargo.toml`, as this requires some special treatments.
 		if top_level_cargo_toml_path == *t {
 			// All workspace member `Cargo.toml` file paths.
-			let workspace_members = cargo_tomls.iter()
-				.filter(|p| **p != top_level_cargo_toml_path)
-				.collect();
+			let workspace_members =
+				cargo_tomls.iter().filter(|p| **p != top_level_cargo_toml_path).collect();
 
 			update_top_level_cargo_toml(&mut cargo_toml, workspace_members, &node_template_path);
 		}
 
 		write_cargo_toml(&t, cargo_toml);
 	});
 
+	// adding root rustfmt to node template build path
+	let node_template_rustfmt_toml_path = node_template_path.join("rustfmt.toml");
+	let root_rustfmt_toml =
+		&options.node_template.parent().unwrap().parent().unwrap().join("rustfmt.toml");
```

Comment:
```suggestion
		&options.node_template.join("../rustfmt.toml");
```

Doesn't this work?

Reply:
I don't think so.  Here's the command that calls this:
```bash
cargo run $PROJECT_ROOT/bin/node-template $PROJECT_ROOT/$PATH_TO_ARCHIVE
```
You'll need to go up two levels from the `options.node_template` arg.  Also, explicity usage of `...` wouldn't be as explicity in error handling if the directory ever moves.  That's okay if it's more legible, but it was at least need to be:
```rust
&options.node_template.join("../../rustfmt.toml");
```
Does that make sense or am I just completely confused here?

Reply:
@danforbes ^^^

Reply:
Sorry, I meant `../../`. I updated the suggestion above!